posts/api/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from .views import PostCreateAPIView, PostDestroyAPIView, PostDetailAPIView, PostListAPIView, PostUpdateAPIView
+
+urlpatterns = [
+    path('', PostListAPIView.as_view(), name='list'),
+    path('create/', PostCreateAPIView.as_view(), name='create'),
+    path('<pk>/', PostDetailAPIView.as_view(), name='detail'),
```

Comment:
Using `pk` won't really be a good option, we need to use `uuidv4` fields to manage posts.

Reply:
The UUID is itself the Pk for the Post model